bin/release_docker_image.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/sh -xe
-
-BASE_TAG=bitjourney/plantuml-service
-TAG=${BASE_TAG}:1.3.3
```

Comment:
@michiomochi Please do not use hard-coded version strings.

Reply:
🙇